src/layout.js

```diff
@@ -505,6 +505,7 @@ export function applyStaticLayout(element) {
     setStyle(element, 'height', dev().assertString(height));
   } else if (layout == Layout.RESPONSIVE) {
     const sizer = element.ownerDocument.createElement('i-amphtml-sizer');
+    sizer.slot = 'i-amphtml-sizer';
```

Comment:
Let's do `setAttribute('slot', '...')`. Less to think about when we work on the SD polyfill.